recipes/vcf2circos/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set name = "vcf2circos" %}
+{% set version = "1.0.1" %}
+{% set sha256 = "3cfd69f722f6f46246f972b917cf46c3af14aa0ccfcd7e029a63f0fb28defd79" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+build:
+  number: 0
+  noarch: generic
+
+source:
+  url: https://github.com/bioinfo-chru-strasbourg/vcf2circos/archive/refs/tags/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+requirements:
+  host:
+    - python >=3.8.0
+    - pandas
+    - plotly
+    - colorlover
+    - dash ==0.39.0
+    - dash-daq ==0.1.0
+    - dash_colorscales
+    - IPython
+    - pyvcf
+    - tqdm
+    - numpy
+    - colour
+    - webcolors
+    - pyfiglet
+    - scipy
+    - python-kaleido
+  run:
+    - python>=3.8.0
```

Comment:
The linter is complaining abut lack of spaces with the version strings

Reply:
Woops 😁